litrpc/lit-accounts.proto

```diff
@@ -186,3 +228,11 @@ message RemoveAccountRequest {
 
 message RemoveAccountResponse {
 }
+
+message AccountIdentifier {
+    // The ID of the account.
+    string id = 1;
+
+    // The label of the account.
```

Comment:
"either the label or the id must be set" right? 

does that mean these can be a "oneof"?

Reply:
Good idea! Addressed in the latest push :)